Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -467,7 +467,9 @@ def make_func(test_spec):
                         def result(self, request, workspace, editor, editor_test_data, launcher_platform):
```

Comment:
Part of the complexity created by all these inline functions is that they're not easy to test. It's quite unclear whether any of the hundreds of lines of highly-mocked unit tests even cover this function. Did you run coverage locally to check whether your new code was covered by tests?

Reply:
Temporarily adding the `assert False` can also quickly prove which tests call your code.

Reply:
Doesn't look like any unit tests are covering this inline function

Reply:
Ideally they could be added for this changed functionality, however I expect it will be difficult to isolate/mock appropriate behavior for a specific unit test. If writing a test is too difficult, I suggest cutting an issue that proposes a refactor of this functionality. Best to not expand the scope of this change too far.

Reply:
I think it would be better to write some integ tests for these instead? I think it would both be easier and add more coverage than unit tests